RELEASING.md

```diff
@@ -25,6 +25,7 @@ These steps are for creating a new major/minor release, with the expectation tha
 
 1. Run release-scripts/do_release.py to copy master directory to the new version directory
 2. Add the new version to `_data/versions.yaml`
+   - Do not add the new version at the top of the file or else it will become the 'Latest Release'
```

Comment:
Can you update this with just a concrete description of where to place it? I'm guessing "it doesn't matter", but let's just pick a location and agree on it to reduce confusion. I'm thinking either second on the list or last on the list.